enterprise/signals.py

```diff
@@ -148,13 +156,17 @@ def assign_or_delete_enterprise_admin_role(sender, instance, **kwargs):     # py
             pending_enterprise_admin_user = None
 
         if kwargs['created'] and pending_enterprise_admin_user:
+            # Create the user in third party analytics application
+            create_tableau_user(instance.user.username, instance)
```

Comment:
Please add details why we are creating tableau user here?

Reply:
Done